File: MoodByte/src/main/java/org/example/ModeloDTO/FraseDTO.java
package org.example.ModeloDTO;

import org.example.Modelo.Frase;

public record FraseDTO(
        long id,
        String frase,
        int puntuacion
) {
    public FraseDTO(Frase frase){
     this(
       frase.getId(),
       frase.getFrase(),
       frase.getPuntuacion()
     );
    }
}

File: MoodByte_App/app/src/main/java/com/example/moodbyte/data/remote/dtos/FraseDto.kt
package com.example.moodbyte.data.remote.dtos

import com.example.moodbyte.data.local.entities.FraseEntity

data class FraseDto(
    val id:Long,
    val frase:String,
    val puntuacion:Int
)

fun FraseDto.toEntity()= FraseEntity(
    id = id,
    frase = frase,
    puntuacion = puntuacion
)